cdc/entry/mounter.go

```diff
@@ -369,7 +367,7 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *TableInfo, row *rowKVEntry) (*m
 		if !exist {
 			return nil, errors.NotFoundf("column info, colID: %d", index)
 		}
-		if !row.Delete && !tableInfo.IsColWritable(colInfo) {
+		if !tableInfo.IsColCDCVisible(colInfo) {
```

Comment:
What is `!row.Delete` here used for originally?

Reply:
The original logic:
if the type of row event here is deletion, the table must be `pk-is-handle`, then we need to output the delete event via rowid, regardless of whether the column is writable or not.

The current logic:
if the table is `pk-is-handle` and the pk column is a virtual generated column, then we won't choose the pk index as handle index;
if the table is `pk-is-handle` and the pk column is a stored generated column, we will output the delete event via rowid, 
so we needn't to judge if the event is delete here